lib/cc/cli/engines.rb

```diff
@@ -9,6 +9,7 @@ module Engines
       autoload :Install, "cc/cli/engines/install"
       autoload :List, "cc/cli/engines/list"
       autoload :Remove, "cc/cli/engines/remove"
+      autoload :Validate, "cc/cli/engines/validate"
```

Comment:
I thought we wanted this to be incorporated into the existing `validate-config` command. Might be a question for the team, both ways make some sense.